src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -649,7 +649,8 @@ def convert_mvs_params_to_epa(mvs_dict, verbatim=False):
                 timeseries = asset[TIMESERIES_SOC].to_list()
                 asset[TIMESERIES_SOC] = {UNIT: unit_soc, VALUE: timeseries}
 
-            if "_excess" not in asset_label and "_sink" not in asset_label:
+            # Excess sinks should not be provided to the EPA
+            if "_excess" not in asset_label:
```

Comment:
@Bachibouzouk is there a reason for filtering out the autogenerated sinks here? Did you only want to remove the excess sinks from being displayed in the EPA, and this was a double-check?

Reply:
Yes this was a double check to not let them be passed to EPA